extension/api.js

```diff
@@ -0,0 +1,80 @@
+      let certDB = Cc["@mozilla.org/security/x509certdb;1"].getService(Ci.nsIX509CertDB);
+      // The last argument is a name which was actually ignored so later removed:
+      // https://dxr.mozilla.org/mozilla-esr52/rev/5350524b/security/manager/ssl/nsIX509CertDB.idl#418
+      certDB.addCertFromBase64(intermediate, ",,", "");
+      console.log("new intermediate certificate added"); // eslint-disable-line no-console
+    } catch (e) {
+      // If this fails, do not move on to the re-verify step.
```

Comment:
should we log a warning or error?

Reply:
yeah, it wasn't in the older hotfix but seems reasonable to log something here, I added it as a console.error.